imap_processing/ultra/l1c/l1c_lookup_utils.py

```diff
@@ -336,7 +343,9 @@ def build_energy_bins() -> tuple[list[tuple[float, float]], np.ndarray, np.ndarr
         Array of geometric means of energy bins.
     """
     # Create energy bins.
-    energy_bin_edges = np.array(UltraConstants.PSET_ENERGY_BIN_EDGES)
+    if energy_bin_edges is None:
+        energy_bin_edges = np.array(UltraConstants.PSET_ENERGY_BIN_EDGES)
```

Comment:
Might want to put a log in here indicating we are using the default.
```suggestion
        energy_bin_edges = np.array(UltraConstants.PSET_ENERGY_BIN_EDGES)
        logger.info(f"No energy bin file found, using default pointing bin energy edges {energy_bin_edges}")
```